components/nothing-found.tsx

```diff
@@ -0,0 +1,24 @@
+import SearchIcon from "@assets/icons/search-icon";
+import { GetStaticProps } from "next";
+import React, { ReactNode } from "react";
+
+type NothingFoundProps = {
+  description: string;
+  action?: ReactNode;
+}
+
+export default function NothingFound({ description, action }: NothingFoundProps) {
+  return (
+      <div className="d-flex flex-column align-items-center gap-4">
+        <SearchIcon />
+        <p className="text-center text-uppercase fw-bold mb-0">{description}</p>
+        { action || null }
```

Comment:
use `children` instead, so we can write it as
```
<NothingFound description="thing">_whatever we want here_</NothingFound>
```
instead of passing html through an input parameter

Reply:
Done. Thanks!